lib/src/views/feed/components/post/post_image.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:feed_sx/src/utils/constants/ui_constants.dart';
+import 'package:flutter/material.dart';
+
+class PostImage extends StatelessWidget {
+  const PostImage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: MediaQuery.of(context).size.width,
```

Comment:
Might be better to make this a function in theme file, since it's being used constantly.

Reply:
Agreed, will make the desired changes.